src/sfProject.ts

```diff
@@ -656,6 +727,34 @@ export class SfProject {
 
     return this.projectConfig;
   }
+
+  public async hasPackageAliases(): Promise<boolean> {
+    return await this.getSfProjectJson().hasPackageAliases();
+  }
+  /**
+   * Returns a read-only list of `packageDirectories` within sfdx-project.json, first reading
+   * and validating the file if necessary. i.e. modifying this array will not affect the
+   * sfdx-project.json file.
+   */
+  public getPackageAliases(): Nullable<Dictionary<string>> {
```

Comment:
this is already defined above

Reply:
The other function is on SfProjectJson. Once again, mirroring the current impl